src/plugins/intel_gpu/src/graph/graph_optimizer/basic_memory_dependencies.cpp

```diff
@@ -36,39 +36,25 @@ void basic_memory_dependencies::run(program& p) {
             add_memory_dependency(it, node);
         }
 
-        if (node->is_type<convolution>() && node->get_preferred_impl_type() == impl_types::onednn) {
-            auto& conv = node->as<convolution>();
-            bool can_reuse_eltwise_mem = false;
+        if (node->get_preferred_impl_type() == impl_types::onednn) {
```

Comment:
Could you be more specific on the list of primitives that support onednn?
That is, only `conv` and `deconv` support sum post op. Others do not support it.

Reply:
I've added a condition to check conv or deconv.